packages/core/package.json

```diff
@@ -23,7 +23,8 @@
   },
   "dependencies": {
     "@logto/connector-kit": "^1.0.0-beta.13",
```

Comment:
better align `*-kit` version? (connector-kit also depends on language-kit)

Reply:
No updates related to the `@logto/connector-kit`, update it when needed.